integration-tests/bats/dump.bats

```diff
@@ -77,6 +77,33 @@ teardown() {
     [[ "$output" = "" ]] || false
 }
 
+@test "dump: SQL type (batched) - compare tables in database with tables imported file " {
```

Comment:
This is a good test but you actually want to examine the file output to see if it's batched.

You might add a --batchSize param to the command so that you can test this more easily (and because it's generally useful)